src/main/kotlin/no/skatteetaten/aurora/gobo/graphql/environment/DeployEnvironmentMutation.kt

```diff
@@ -0,0 +1,61 @@
+package no.skatteetaten.aurora.gobo.graphql.environment
+
+import com.expediagroup.graphql.server.operations.Mutation
+import graphql.schema.DataFetchingEnvironment
+import java.util.Date
+import no.skatteetaten.aurora.gobo.graphql.token
+import no.skatteetaten.aurora.gobo.integration.phil.PhilResult
+import no.skatteetaten.aurora.gobo.integration.phil.PhilService
+import org.springframework.stereotype.Component
+
+@Component
+class DeploymentEnvironmentMutation(
+    private val philService: PhilService,
+) : Mutation {
+    suspend fun deployEnvironment(
+        input: DeploymentEnvironmentInput,
+        dfe: DataFetchingEnvironment
+    ): DeploymentEnvironmentResponse {
+        val philDeploymentResult = philService.deployEnvironment(input.environment, dfe.token())
+        return philDeploymentResult.toDeploymentEnvironmentResponse()
```

Comment:
Nå har vi 2 * Deployment og DeploymentRef. Kan vi bare bruke verdien vi får fra Phil og returnere?

Reply:
Jeg er aldri noe glad i å direktekoble variable mellom to APIer/grensesnitt. Om vi returnerer objektene fra Phil direkte, så får vi dessuten med status (som var hovedgrunnen til å frikoble, og som illustrerer denne avhengigheten godt).

Reply:
På den annen side, så kan vi si at Gobo alltid skal returnere det man får fra Phil, altså at definisjonen for deployEnvironment-endepunktet til Gobo er å sende videre til Phil og returnere det som kommer tilbake.  I så fall kan man bare fjerne, men man må da også være klar over at man åpner mer for at Gobos API-endepunkt for dette vil endre kontrakt dersom Gobo-Phil kommunikasjonen endrer seg.

Reply:
Etter diskusjon: Renavner til *Resource i integrasjonsobjektene.